app/models/lightning.py

```diff
@@ -278,6 +278,43 @@ def from_grpc(cls, h) -> "RouteHint":
         return cls(hop_hints=hop_hints)
 
 
+class Channel(BaseModel):
+
+    channelID: Optional[str]
+    active: Optional[bool]
+
+    peerPublicKey: Optional[str]
+    peerAlias: Optional[str]
+
+    balanceLocal: Optional[int]
+    balanceRemote: Optional[int]
+    balanceCapacity: Optional[int]
+
+    @classmethod
+    def from_grpc(cls, c) -> "Channel":
+        return cls(
+            active=c.active,
+            channelID=c.channel_point, # use channel point as id because thats needed for closing the channel with lnd
+            peerPublicKey=c.remote_pubkey,
+            peerAlias="n/a",
+            balanceLocal=c.local_balance,
+            balanceRemote=c.remote_balance,
+            balanceCapacity=c.capacity
+        )
+
+    @classmethod
+    def from_grpc_pending(cls, c) -> "Channel":
+        print(str(c))
```

Comment:
Probably left accidentally?

Reply:
thats needed and in use ... its to parse channel info from grpc opening pending channels objects, that are slightly different from already active channels - so a special parsing method was needed

Reply:
The print statement?

Reply:
ah Ok - thats gone now